charts/icm-web/templates/cluster-pagecache-pvc.yaml

```diff
@@ -0,0 +1,17 @@
+{{- if and (.Values.persistence.cluster.enabled) (not .Values.persistence.cluster.existingClaim) -}}
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: {{ include "icm-web.fullname" . }}-cluster-pagecache-pvc
+spec:
+  accessModes:
+    - ReadWriteMany
+  {{- if .Values.persistence.cluster.storageClass }}
```

Comment:
There is function `coalesce` in Helms Sprig library -> http://masterminds.github.io/sprig/defaults.html to safe yourself this if-else-end

Reply:
Could idea with `coalesce`, but I decided to remove the if statement since it's not needed. The storage classname could be just configured.